src/Storefront/Resources/views/storefront/page/account/sidebar.html.twig

```diff
@@ -76,8 +76,8 @@
                                 <a href="{{ path('frontend.account.logout.page') }}"
                                    data-account-guest-abort-button="true"
                                    class="btn btn-link account-aside-btn">
-                                    {% sw_icon 'log-out' %}
-                                    {{ 'account.guestAbort'|trans|sw_sanitize }}
+                                    {# Do not add a space, as the underline text-decoration will show this #}
```

Comment:
Could we also use `{%- -%}` to the surrounding block to get rid of the [whitespace](https://twig.symfony.com/doc/3.x/templates.html#whitespace-control)?

Reply:
Yes, but we need to add it to the inner elements, where the whitespace should be removed :-) But that works.